### PR TITLE
completed styling and components

### PR DESCRIPTION
**Pull request by ccades**

### CODE REVIEW

File: database/transaction.js [hunk 1]
```diff
@@ -60,31 +71,37 @@ const transactionSchema = new mongoose.Schema({
...
+    date: getDate(),
+    time: getTime(),
     quantity: transaction.quantity,
-    status: 'pending',
+    status: 'Queued',
     filled: 'TBD',
     filledQuantity: 0,
     totalCost: 0.00,
-  };
-  console.log(newTx);
```
Comment (morgangrace11) [inline database/transaction.js:81]: Maybe just taking out console logs that aren't needed anymore

File: server/index.js [hunk 2]
```diff
@@ -2,15 +2,16 @@ const express = require('express');
...
 const port = 3333;
 
-app.use(express.static('public'));
+app.use('/', express.static('./public'));
+app.use(/\/\d+\//, express.static('./public'));
+
 app.use(bodyParser.json());
 
 app.get('/stocks/:stockTicker', (req, res) => {
-  console.log('PARAMMMMSSSSSS', req.params);
```
Comment (morgangrace11) [inline server/index.js:13]: Alos this console log

**Review by morgangrace11 — APPROVED** [review]
lgtm! Just a couple of stray console.logs that i'd say get rid of!

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -1,17 +1,242 @@
 import React from 'react';
+import BuySell from './BuySell.jsx';
+import History from './History.jsx';
```
Comment (therobinkim) [inline client/src/components/App.jsx:3]: I like to list npm modules first, and then my own modules last

File: client/src/components/App.jsx [hunk 4]
```diff
@@ -1,17 +1,242 @@
...
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      transactions: []
     }
   }
 
+  componentWillMount() {
```
Comment (therobinkim) [inline client/src/components/App.jsx:208]: you should not use `componentWillMount`. are you looking at old documentation/tutorials?
use `componentDidMount` instead
https://reactjs.org/docs/react-component.html#legacy-lifecycle-methods

File: client/src/components/BuySell.jsx [hunk 5]
```diff
@@ -0,0 +1,1341 @@
...
+height: 19px;
+letter-spacing: 0.25px;
+line-height: 19px;
+text-align: left;
+text-size-adjust: 100%;
+white-space: nowrap;
+width: 45px;
+-webkit-box-direction: normal;
+-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+`;
```
Comment (therobinkim) [inline client/src/components/BuySell.jsx:1092]: where is all of this CSS coming from? did you write it all? it seems a little crazy that we have 1100 lines of CSS in this JS file

File: client/src/components/BuySell.jsx [hunk 6]
```diff
@@ -0,0 +1,1341 @@
...
+  }
+
+  changeStock() {
+    axios.get('/stocks').then(res => {
+      const stocks = res.data;
+      const id = parseInt(window.location.pathname.slice(1, window.location.pathname.length-1))
+      var urlStock = {}
+      for (var i=0; i < stocks.length; i++) {
+        if (stocks[i]._id === id) {
+            urlStock = stocks[i];
```
Comment (therobinkim) [inline client/src/components/BuySell.jsx:1152]: fix indentation

File: client/src/components/BuySell.jsx [hunk 7]
```diff
@@ -0,0 +1,1341 @@
...
+      borderRightColor: 'rgb(33, 206, 153)'
+    })
+  }
+
+  sharesBlur() {
+    this.setState({
+      borderTopColor: 'rgb(23, 23, 24)',
+      borderBottomColor: 'rgb(23, 23, 24)',
+      borderLeftColor: 'rgb(23, 23, 24)',
+      borderRightColor: 'rgb(23, 23, 24)'
```
Comment (therobinkim) [inline client/src/components/BuySell.jsx:1200]: seems like these strings are being repeated an awful lot.. can we just store these strings elsewhere and instead set state to strings like "dark gray", "light gray", "yellow" (which then reference these specific RGB values)?

File: client/src/components/BuySell.jsx [hunk 8]
```diff
@@ -0,0 +1,1341 @@
...
+    }
+  }
+
+  handleChange(e) {
+    e.preventDefault();
+    var cost = (e.target.value * this.state.currentPrice)
+    this.setState({
+      shares: e.target.value,
+      estimatedCost: cost.toFixed(2)
+    })
```
Comment (therobinkim) [inline client/src/components/BuySell.jsx:1229]: ;
do you have a linter installed?

File: client/src/components/BuySell.jsx [hunk 9]
```diff
@@ -0,0 +1,1341 @@
...
+        sharesBalance: `${this.state.userBalance} Buying Power Available`,
+        isSelectedBuy: true,
+        isSelectedSell: false
+      })
+    }
+  }
+
+  handleChange(e) {
+    e.preventDefault();
+    var cost = (e.target.value * this.state.currentPrice)
```
Comment (therobinkim) [inline client/src/components/BuySell.jsx:1225]: ;
do you have a linter installed?

File: client/src/components/BuySell.jsx [hunk 10]
```diff
@@ -0,0 +1,1341 @@
...
+        isSelectedBuy: false,
+        isSelectedSell: true
+      })
+    } else {
+      this.setState({
+        orderDescription: 'Estimated Cost',
+        sharesBalance: `${this.state.userBalance} Buying Power Available`,
+        isSelectedBuy: true,
+        isSelectedSell: false
+      })
```
Comment (therobinkim) [inline client/src/components/BuySell.jsx:1219]: ;
do you have a linter installed?

File: client/src/components/BuySell.jsx [hunk 11]
```diff
@@ -0,0 +1,1341 @@
...
+
+  buySellClick(e) {
+    console.log('CLICK!!!!', e.target.id, 'BUY', this.state.isSelectedBuy, 'SELL', this.state.isSelectedSell)
+    if (e.target.id === 'sell') {
+      this.setState({
+        orderDescription: 'Estimated Credit',
+        sharesBalance: `${this.state.userShares} Shares Available`,
+        isSelectedBuy: false,
+        isSelectedSell: true
+      })
```
Comment (therobinkim) [inline client/src/components/BuySell.jsx:1212]: ;
do you have a linter installed?

File: client/src/components/BuySell.jsx [hunk 12]
```diff
@@ -0,0 +1,1341 @@
...
+    var cost = (e.target.value * this.state.currentPrice)
+    this.setState({
+      shares: e.target.value,
+      estimatedCost: cost.toFixed(2)
+    })
+  }
+
+  buyStock(e) {
+
+    const orderType = this.state.isSelectedBuy ? 'Market Buy' : 'Market Sell'
```
Comment (therobinkim) [inline client/src/components/BuySell.jsx:1234]: ;
do you have a linter installed?

File: public/bundle.js [hunk 13]
```diff
@@ -94,7 +94,43 @@
 /***/ (function(module, __webpack_exports__, __webpack_require__) {
 
 "use strict";
```
Comment (therobinkim) [inline public/bundle.js:96]: don't commit your bundle

File: server/index.js [hunk 14]
```diff
@@ -2,15 +2,16 @@ const express = require('express');
...
 const db = require('../database/index.js');
-const transactions = require('../database/Transaction.js');
-const stocks = require('../database/Stock.js');
+const transactions = require('../database/transaction.js');
+const stocks = require('../database/stock.js');
 const port = 3333;
 
-app.use(express.static('public'));
+app.use('/', express.static('./public'));
+app.use(/\/\d+\//, express.static('./public'));
```
Comment (therobinkim) [inline server/index.js:10]: what does this regex do?

File: server/index.js [hunk 15]
```diff
@@ -20,12 +21,33 @@ app.get('/stocks/:stockTicker', (req, res) => {
   });
 });
 
+app.get('/stocks', (req, res) => {
+  stocks.loadAllStocks((err, data) => {
+    if (err) {
```
Comment (therobinkim) [inline server/index.js:26]: `err` never exists according to your implementation of `loadAllStocks`. is this conditional necessary?

File: server/index.js [hunk 16]
```diff
@@ -20,12 +21,33 @@ app.get('/stocks/:stockTicker', (req, res) => {
...
+      res.json({ message: 'The market is no more' });
+    } else {
+      res.json(data);
+    }
+  });
+});
+
+app.get('/transactions', (req, res) => {
+  transactions.loadAll((err, data) => {
+    if (err) {
```
Comment (therobinkim) [inline server/index.js:36]: `err` never exists according to your implementation of `loadAll`. is this conditional necessary?

**Pull request merged**